build.cmd

```diff
@@ -484,7 +487,7 @@ if /i "%__DoCrossArchBuild%"=="1" (
     set __MsbuildLog=/flp:Verbosity=normal;LogFile=!__BuildLog!
     set __MsbuildWrn=/flp1:WarningsOnly;LogFile=!__BuildWrn!
     set __MsbuildErr=/flp2:ErrorsOnly;LogFile=!__BuildErr!
-
+    
```

Comment:
This looks like an extraneous whitespace change

Reply:
Yeah, will fix it.